### PR TITLE
Feature/removing enums and using ros message  values instead

### PR DESCRIPTION
**Pull request by sanazfth**
- removing enums and using ros message  values instead
- uint16 TURN_HAZARDS = 3 needs to be added to PacmodCmd.msg 
https://github.com/astuff/astuff_sensor_msgs/pull/48

### CODE REVIEW

File: src/publish_control_board_rev2.cpp [hunk 1]
```diff
@@ -74,27 +74,27 @@ void PublishControlBoardRev2::publish_turn_signal_message(const sensor_msgs::Joy
     // Axis 2 is the "left trigger" and axis 5 is the "right trigger" single
     // axis joysticks on the back of the controller
     if (msg->axes[2] < -0.5)
-      turn_signal_cmd_pub_msg.ui16_cmd = SIGNAL_HAZARD;
+      turn_signal_cmd_pub_msg.ui16_cmd = pacmod_msgs::PacmodCmd::TURN_HAZARDS;
```
Comment (icolwell-as) [inline src/publish_control_board_rev2.cpp]: Should this be `pacmod_msgs::SystemRptInt::TURN_HAZARDS` instead?

**Comment by icolwell-as** [comment]
> uint16 TURN_HAZARDS = 3 needs to be added to PacmodCmd.msg

Can you link the PR that is making that addition?

**Comment by sanazfth** [comment]
[https://github.com/astuff/astuff_sensor_msgs/pull/48](url)

**Comment by JWhitleyAStuff** [comment]
CI fixed.

**Pull request merged**